dotnet/samples/HomeAutomation/README.md

```diff
@@ -0,0 +1,40 @@
+# "House Automation" example illustrating how to use Semantic Kernel with dependency injection
+
+This example demonstrates a few dependency injection patterns that can be used with Semantic Kernel.
+
```

Comment:
Once we turn the sample into a turn-based application, you can add the recommended questions to ask the AI here.

Reply:
Questions have been suggested in app